.github/workflows/version.yml

```diff
@@ -33,8 +33,8 @@ jobs:
       - uses: actions/upload-artifact@v2
         name: Upload Extension Zip
         with:
-          name: connect-extension
-          path: connect-extension.zip
+          name: stacks-wallet-chromium
+          path: stacks-wallet-chromium.zip
```

Comment:
+1

Reply:
This file is then uploaded to GCS, which is how it's downloaded from hiro.so. We'd have to duplicate the .zip and then upload the version without a filename.

There is more work required to update our actual production flows, because right now it's quite tied to the app.blockstack.org production flow. I'm going to table this, it's a good suggestion, but I'd rather combine it with other CI work.